server/middlewares/error-handlers.js

```diff
@@ -22,7 +17,11 @@ export default function prodErrorHandler() {
     // parse res type
     const accept = accepts(req);
     const type = accept.type('html', 'json', 'text');
-    const handled = unwrapHandledError(err);
+
+    // If err is array then use the unwrap function,
+    // otherwise it wasn't wrapped so we build the handled object.
+    const handled = err.constructor === Array ? unwrapHandledError(err) :
+      { message: err.message, stack: err.stack };
```

Comment:
Why would the error be an array? Also, a better check for type array is  `Array.isArray()`.